tests/acceptance/features/webUILogin/login.feature

```diff
@@ -25,14 +26,24 @@ Feature: login users
     When the user logs out of the webUI
     Then the authentication page should be visible
 
+  Scenario: logging out redirects to the url with state attribute
+    Given these users have been created with default attributes:
+      | username |
+      | user1    |
+    And user "user1" has logged in using the webUI
+    And the user has browsed to the files page
+    When the user logs out of the webUI
+    Then the redirected url should contain "/#/login?state="
+
+  @skip
```

Comment:
please add an issue, why this is skipped

Reply:
I unskipped the test since I figured out the issue. The issue has been tagged and expected behavior has been commented on the scenario.